examples/solids/Makefile

```diff
@@ -79,11 +79,11 @@ $(PETSc.pc):
 pkgconf = $(shell pkg-config $1 | sed -e 's/^"//g' -e 's/"$$//g')
 
 # E = 2 * mu + (1 + nu)
-output/NH-strain.csv: args = -problem FSInitial-NH1 -E 2.8 -nu 0.4
-output/MR-strain.csv: args = -problem FSInitial-MR1 -mu_1 1 -mu_2 0.0 -nu .4
-output/MR-strain1.csv: args = -problem FSInitial-MR1 -mu_1 .5 -mu_2 0.5 -nu .4
+tests-output/NH-strain.csv: args = -problem FSInitial-NH1 -E 2.8 -nu 0.4
+tests-output/MR-strain.csv: args = -problem FSInitial-MR1 -mu_1 1 -mu_2 0.0 -nu .4
+tests-output/MR-strain1.csv: args = -problem FSInitial-MR1 -mu_1 .5 -mu_2 0.5 -nu .4
 
-output/%.csv: elasticity
+tests-output/%.csv: elasticity
```

Comment:
These files are referenced from the Sphinx documentation.

Reply:
Good catch, thank you